src/model/ControllerOutdoorAir_Impl.hpp

```diff
@@ -67,8 +68,10 @@ namespace model {
       boost::optional<double> getEconomizerMaximumLimitDewpointTemperature() const;
       bool setEconomizerMaximumLimitDewpointTemperature(boost::optional<double> value);
 
-      //QuadraticCurve getElectronicEnthalpyLimitCurve() const;
-      //bool setElectronicEnthalpyLimitCurve(QuadraticCurve c);
+      boost::optional<Curve> getElectronicEnthalpyLimitCurve() const;
+      boost::optional<Curve> electronicEnthalpyLimitCurve() const;
```

Comment:
On the Impl side, I **definitely** want just the one gettter.

```suggestion
      boost::optional<Curve> electronicEnthalpyLimitCurve() const;
```